charts/ChartConfig.tsx

```diff
@@ -238,6 +244,9 @@ export class ChartConfigProps {
     @observable.ref tab: ChartTabOption = "chart"
     @observable.ref overlay?: ChartTabOption = undefined
 
+    @observable relatedQuestions: RelatedQuestionsConfig[] = [
```

Comment:
should make this field optional (`relatedQuestions?`) just like almost all other fields. This is also why Travis is failing right now.

Reply:
Right, I went back and forth with this as this was too my preference but what I had tried until now was making things unnecessarily obscure. It should be better now.